experts-migration-commands.php

```diff
@@ -0,0 +1,742 @@
+					"label"       => "SlideShare",
+					"service_url" => "http://www.slideshare.net/",			
+					"user_url"    => "http://www.slideshare.net/{value}" ),
+		);
+		foreach( $social_media_options as $option ){
+			if ( $social_media['option'] == $option['label'] ) {
+				return '<!-- wp:paragraph --><p><a href="'.str_replace( '{value}', $social_media['username'], $option['user_url'] ).'"><strong>'.$option['label'].'/ </strong> '.$social_media['username'].'</a></p><!-- /wp:paragraph -->';
+			}
+		}	
+		
+	}
+	
+	/** 
+	* ## OPTIONS
+	*
+	* <page>
+	* : The page of the query we are in.
+	*
+	* [--page=<page>]
+	* : What page in the query we are processing.
+	* ---
+	* default: 1
+	* ---
+	*
+	* ## EXAMPLES
+	*
+	*     wp bam_ubc_experts process_experts 3
+	*
+	*/
+	public function process_experts( $args, $assoc_args ) {
+	
+		$gutenberg_template = '<!-- wp:columns -->
+	<div class="wp-block-columns"><!-- wp:column {"width":"33.33%%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%%">
+	<!-- wp:image {"id":10000,"sizeSlug":"full","linkDestination":"none"} -->
+	<figure class="wp-block-image size-full"><img src="%s" alt="" class="wp-image-10000"/></figure>
+	<!-- /wp:image --></div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"66.66%%"} -->
+	<div class="wp-block-column" style="flex-basis:66.66%%"><!-- wp:heading -->
+	<h2>%s</h2>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3>%s</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":4} -->
+	<h4>%s</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3><br>Fields</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph -->
+	<p>%s</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:heading {"level":3} -->
+	<h3><br>Expertise</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph -->
+	<p>%s</p>
+	<!-- /wp:paragraph -->
+	
+	
+	</div>
+	<!-- /wp:column --></div>
+	<!-- /wp:columns -->
+	
+	<!-- wp:columns -->
+	<div class="wp-block-columns">
+	<!-- wp:column {"width":"33.33%%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%%">
+	<!-- wp:heading {"level":3} -->
+	<h3>Contact Information</h3>
+	<!-- /wp:heading -->
+	%s 
+	
+	%s 
+	
+	%s
+	
+	%s
+	
+	%s
+	
+	%s
+	
+	<!-- wp:block {"ref":%d} /-->
+	</div>
+	<!-- /wp:column -->
+	
+	<!-- wp:column {"width":"66.66%%"} -->
+	<div class="wp-block-column" style="flex-basis:66.66%%">
+	
+	%s
+	
+	%s 
+	
+	%s
+	
+	%s
+	</div>
+	<!-- /wp:column -->
+	</div>
+	<!-- /wp:columns -->';
+	
+		$paged = ( $args[0] ) ? $args[0] : 1;
+	
+		$profile_args = array(
+			'post_type'			=> 'profile_cct',
+			'posts_per_page'    => 30,
+			'paged' 			=> $paged
+		);
+		
+		$profiles = new WP_Query($profile_args);
+		if ( $profiles->have_posts() ) {
+	
+			// wp_block
+			while( $profiles->have_posts() ){
+				$profiles->the_post();
+				$post_id = get_the_ID();
```

Comment:
Brilliantly, `get_the_ID()` doesn't escape the data it returns. And it can also return `false`. So it's safest to run this through `absint()` and check the value is not 0 (or other appropriate ways to validate and sanitize)